docs/src/pages/Tutorial/tutorial.md

```diff
@@ -80,7 +80,7 @@ app we want to build, with this HTML:
 
 It looks like this:
 
-![what it looks like](./tut1.png)
+![what it looks like](https://github.com/jorgebucaran/hyperapp/blob/1fd42319051e686adb9819b7e154f764fa3b0d29/docs/src/pages/Tutorial/tut1.png?raw=true)
```

Comment:
@zaceno Wouldn't it better to use the raw URL directly?

`https://raw.githubusercontent.com/jorgebucaran/hyperapp/1fd42319051e686adb9819b7e154f764fa3b0d29/docs/src/pages/Tutorial/tut1.png`

Reply:
Yes it would 😄 ! I just didn't know. fixed it now